corehq/apps/geospatial/templates/case_grouping_map.html

```diff
@@ -88,12 +88,12 @@
         </div>
         <div class="row">
           <div class="col col-md-6">
-            <button class="btn-default form-control" data-bind="click: showSelectedGroups()">
+            <button class="btn-default form-control" data-bind="click: showSelectedGroups(), disable: !groupsReady()">
```

Comment:
This looks like it might give a ko exception when executed. The `exportModel` ko model has been binded to the `export-controls` element, but this button element is currently not in scope for that element. This button wouldn't have access to `groupsReady()` from `exportModel`.

Reply:
`groupsReady` is defined on `exportModel` and `caseGroupSelectModel`

Reply:
Ah! It seems I completely missed that, thanks for clarifying.